application/common/encryption.py

```diff
@@ -0,0 +1,102 @@
+from aws_encryption_sdk import CommitmentPolicy, EncryptionSDKClient, StrictAwsKmsMasterKeyProvider
+from aws_encryption_sdk.exceptions import (
+    MaxEncryptedDataKeysExceeded,
+    NotSupportedError,
+    SerializationError,
+    UnknownIdentityError,
+)
+from aws_lambda_powertools import Logger
+from aws_lambda_powertools.middleware_factory import lambda_handler_decorator
+from aws_lambda_powertools.utilities.typing import LambdaContext
+from base64 import b64decode
+from binascii import Error as binasciiError
+from boto3 import client
+from json import loads, dumps
+from os import environ
+from time import time
+from typing import Any, Dict
+
+from common.utilities import extract_body
+
+logger = Logger(child=True)
+
+SIGNING_KEY_KEY = "signing_key"
+BAD_REQUEST = {"statusCode": 400, "body": dumps({"message": "Invalid payload"})}
+
+
+def initialise_encryption_client():
```

Comment:
Why is this a function that returns two functions, rather than a class that has two function you can call ?

Reply:
no reason other than that ive spent last 4 years working in Typescript lambda and react hooks where ive done everything functional

Reply:
cool, agreed to look at changing this in a future refactor for clean code purposes, no point changing it at the moment considering it's functionality fine